pages/en/lb3/Security-advisory-03-10-2017.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Security advisory 03-10-2017"
+lang: en
+layout: page
+toc: false
+keywords: LoopBack
+tags: security
+sidebar: lb3_sidebar
+permalink: /doc/en/lb3/Security-advisory-03-10-2017.html
+---
+*   **Security risk**: Medium-high
+*   **Vulnerability**: `loopback-component-storage` to directory traversal attack
+
+### Description
+
+A security leak exposing `loopback-component-storage` to directory traversal attack. The component was exposed to a vulnerability where an attacker could use a command to retrieve the content of the `server.js` file of a LoopBack application and crash the server.
+
+### Reported by
+
+Pratibha Rikh
+
+### Versions affected
+
+loopback-component-storage 3.0.0 and earlier
+
+### Solution
+
+Upgrade to loopback-component-storage  3.0.1 or later.
+
+Ensure that your application's `package.json` has the following line:
+
+```js
+"dependencies": {
+   ...
+  "loopback-component-storage": "^3.0.0",
```

Comment:
Loay agreed that  ^3.0.0 will fetch the most recent release within the major version, which is 3.2.0, so it's good "as is."